src/Tags/Tagging.hs

```diff
@@ -29,30 +30,41 @@ runTagging blob symbolsToSummarize
   = Eff.run
   . evalState @[ContextToken] []
   . Streaming.toList_
-  . contextualizing blob symbolsToSummarize
+  . contextualizing blob toKind
   . tagging blob
+  where
+    toKind x = do
+      guard (x `elem` symbolsToSummarize)
+      case x of
```

Comment:
This `case` statement should be the other end of the `APIBridge` instance that I mentioned above.